core/src/main/resources/hudson/model/View/index.jelly

```diff
@@ -44,11 +44,24 @@ THE SOFTWARE.
         <l:header>
             <!-- for screen resolution detection -->
             <l:yui module="cookie" />
-            <script>
-              YAHOO.util.Cookie.set("screenResolution", screen.width+"x"+screen.height, {
-                path: "/"
-              });
-            </script>
+            <j:invokeStatic var="req" className="org.kohsuke.stapler.Stapler" method="getCurrentRequest"/>
```

Comment:
As a rule, if you find the need to use `j:invokeStatic`, it is a good sign you should be pushing the computation into the controller (`View.java`).

In this case, IIRC `request` or something like it is already bound in page variables.